lib/utils/interceptor.d.ts

```
@@ -1,6 +1,7 @@
-export declare type Interceptor = (...args: any[]) => Promise<boolean> | boolean | undefined | void;
-export declare function callInterceptor(interceptor: Interceptor | undefined, { args, done, canceled, }: {
+export type Interceptor = (...args: any[]) => Promise<boolean> | boolean | undefined | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)